tools/tokenserver/test_purge_old_records.py

```diff
@@ -0,0 +1,136 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file was adapted from [this file](https://github.com/mozilla-services/tokenserver/blob/master/tokenserver/tests/test_purge_old_records.py) on the legacy Tokenserver